File: Module.Frontend.TPM/Client/model/tpm/costproduction/HistoricalCostProduction.js
Ext.define('App.model.tpm.costproduction.HistoricalCostProduction', {
    extend: 'Ext.data.Model',
    idProperty: '_Id',
    breezeEntityType: 'PromoSupport',
    fields: [
        { name: '_Id', type: 'string', hidden: true },
        { name: '_ObjectId', hidden: true },
        { name: '_User', type: 'string', isDefault: true },
        { name: '_Role', type: 'string', isDefault: true },
        { name: '_EditDate', type: 'date', isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        { name: '_Operation', type: 'string', isDefault: true },
        { name: 'PlanQuantity', type: 'int', hidden: false, isDefault: true },
        { name: 'ActualQuantity', type: 'int', hidden: false, isDefault: true },
        { name: 'PlanCostTE', type: 'float', hidden: false, isDefault: true },
        { name: 'ActualCostTE', type: 'float', hidden: false, isDefault: true },
        { name: 'StartDate', type: 'date', hidden: false, isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        { name: 'EndDate', type: 'date', hidden: false, isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        { name: 'PlanProdCostPer1Item', type: 'float', hidden: true, isDefault: false },
        { name: 'ActualProdCostPer1Item', type: 'float', hidden: true, isDefault: false },
        { name: 'PlanProdCost', type: 'float', hidden: true, isDefault: false },
        { name: 'ActualProdCost', type: 'float', hidden: true, isDefault: false },
        { name: 'AttachFileName', type: 'string', hidden: true, isDefault: false },
        { name: 'PONumber', type: 'string', hidden: false, isDefault: true },
        { name: 'InvoiceNumber', type: 'string', hidden: false, isDefault: true },
        { name: 'ClientTreeFullPathName', type: 'string', hidden: false, isDefault: true },
        { name: 'BudgetSubItemName', type: 'string', hidden: false, isDefault: true },
        { name: 'BudgetSubItemBudgetItemName', type: 'string', hidden: false, isDefault: true },
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'HistoricalPromoSupports',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        },
        extraParams: {
            Id: null
        }
    }
});

File: Module.Persist.TPM/Model/History/HistoricalCostProduction.cs
using Core.History;
using System;

namespace Module.Persist.TPM.Model.History
{
    public class HistoricalCostProduction : BaseHistoricalEntity<Guid>
    {
        public int Number { get; set; }
        public int? PlanQuantity { get; set; }
        public int? ActualQuantity { get; set; }
        public double? PlanCostTE { get; set; }
        public double? ActualCostTE { get; set; }
        public DateTimeOffset? StartDate { get; set; }
        public DateTimeOffset? EndDate { get; set; }
        public double? PlanProdCostPer1Item { get; set; }
        public double? ActualProdCostPer1Item { get; set; }
        public double? PlanProdCost { get; set; }
        public double? ActualProdCost { get; set; }
        public string AttachFileName { get; set; }
        public string BorderColor { get; set; }
        public string PONumber { get; set; }
        public string ClientTreeFullPathName { get; set; }
        public string BudgetItemName { get; set; }
        public string BudgetSubItemBudgetItemName { get; set; }
    }
}